Syntaxes/HTML (Django).sublime-syntax

```diff
@@ -0,0 +1,241 @@
+%YAML 1.2
+---
+# https://www.sublimetext.com/docs/syntax.html
+# https://html.spec.whatwg.org/multipage/syntax.html
+name: HTML (Django) Update
+scope: text.html.django
+version: 2
+
+extends: Packages/HTML/HTML.sublime-syntax
+
+file_extensions:
+  - html
+  - htm
+  - shtml
+  - xhtml
+
+# match on "{% load " or "{% extends "
+first_line_match: (?i){%\s?(?>load|extends)\s
+
+variables:
+  template_tags: "(
```

Comment:
Would be possible to match whole words only? This doesn't work well in some cases: https://i.imgur.com/5XzkGOo.png

Reply:
If I add `- match: '\b{{template_tags}}\b'` (with single quotes) that does the whole word matching for me but only if I also make the value of `template_tags` a single line string. Otherwise only words that are the last one on each line (in the definition) are highlighted, e.g. `endfor` matches but not `for`. This is with ST3.

Reply:
Actually, `- match: '\b{{template_tags}}\b'` does work just fine! 

(The problem for me was that I also made the definition of `template_tags` use single quotes, but that has to remain with double quotes. I don't know how this works exactly.)

Reply:
Thank you for putting this together! However, the [current version](https://github.com/squ1b3r/Djaneiro/blob/4e91987b689702260afc2f471a32441d2f4d08c6/Syntaxes/HTML%20(Django).sublime-syntax) now doesn't work at all: https://i.imgur.com/ruCH0sT.png I still only tried on ST3.

Reply:
Looks like this is due to using a pop command that supports only build >4050